### PR TITLE
Hotfix/cdap 3.0.6 tephra 0.4.4

### PR DESCRIPTION
**Pull request by sagarkapare**

### CODE REVIEW

**Comment by anwar6953** [comment]
This PR also resolves the following JIRA for the release/3.0 branch (CDAP 3.0.6):
https://issues.cask.co/browse/CDAP-4067

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -377,14 +377,13 @@
 
     <property>
         <name>data.tx.client.count</name>
-        <value>5</value>
-        <description>The number of pooled instanced of the transaction
-            client</description>
+        <value>50</value>
+        <description>The number of pooled instanced of the transaction client</description>
```
Comment (sreevatsanraman) [inline cdap-common/src/main/resources/cdap-default.xml]: Add some notes on what's the impact on changing it.

**Comment by poornachandra** [comment]
@sreevatsanraman Updated description, can you take another look?

**Comment by sreevatsanraman** [comment]
lgtm

**Pull request merged**